rlbot_gui/bot_management/downloader.py

```diff
@@ -43,3 +46,19 @@ def download_gitlfs(repo_url: str, checkout_folder: Path, branch_name: str):
     file.close()
 
     print('Done downloading git repo.')
+
+def download_botfs(repo_url: str, checkout_folder: Path, branch_name: str, bot_path: str):
+    print('Starting download of a git repo... this might take a while.')
```

Comment:
can we use the logger instead of print?

Reply:
here and everywhere else

Reply:
can. was just an alpha way of displaying the start and end moment of downloading the repo